netlify.toml

```diff
@@ -0,0 +1,19 @@
+# This should be used for docs also
+[[headers]]
+  for = "/*"
+  [headers.values]
+    Content-Security-Policy = "default-src https:; script-src https: 'unsafe-inline'; style-src https: 'unsafe-inline'; img-src https: data: blob:"
+    Permissions-Policy = "geolocation=(),midi=(),sync-xhr=(self),microphone=(),camera=(),magnetometer=(),gyroscope=(),fullscreen=(self),payment=()"
+    Referrer-Policy = "no-referrer"
+    X-Frame-Options = "SAMEORIGIN"
+    X-Xss-Protection = "1; mode=block"
+    X-Content-Type-Options = "nosniff"
+
+[[headers]]
+  for = "/-/*"
+  [headers.values]
+    cache-control = '''
+    max-age=0,
+    no-cache,
+    no-store,
+    must-revalidate'''
```

Comment:
I don't think this is needed. The application is a SPA, so it serves static content on all routes. Any dynamic parameters in the query string will only be consumed client-side, not rendered in the content.

Reply:
ok, we can remove this as its not working as expected.